app/decorators/ncr/work_order_decorator.rb

```diff
@@ -13,6 +13,14 @@ def final_approver_email_address
       approver_email_address(final_approver)
     end
 
+    def status_aware_approver_email_address
```

Comment:
I know we have the `reporter` spec below, but what about a unit test for this to explain reasoning behind logic? If I were going to update this decorator, I would assume it wasn't covered by tests because there is no unit test.